.gitignore

```diff
@@ -6,7 +6,6 @@
 
 # Typescript v1 declaration files
 typings/
-
 node_modules/
-
 secrets/
+.vscode/settings.json
```

Comment:
why publish vscode-settings?

Reply:
it's just not being published is being ignored?!